flink-runtime/src/main/java/org/apache/flink/runtime/blob/TransientBlobCleanupTask.java

```diff
@@ -100,9 +102,15 @@ public void run() {
 				writeLock.lock();
 
 				try {
-					if (!localFile.delete() && localFile.exists()) {
-						log.warn("Failed to locally delete blob " + localFile.getAbsolutePath());
-					} else {
+					try {
+						Files.delete(localFile.toPath());
+					} catch (IOException e) {
```

Comment:
Why are `IOException` and `Exception` caught separately if the action is the same?